src/plone/base/interfaces/controlpanel.py

```diff
@@ -1332,6 +1332,18 @@ class ISiteSchema(Interface):
         value_type=schema.Choice(vocabulary="plone.app.vocabularies.Roles"),
     )
 
+    use_ajax = schema.Bool(
```

Comment:
use_ajax_main_template would be a clearer name.

Reply:
:thinking:

Reply:
@davisagli sorry, wrong icon. I case you wondered about this reaction, this face 
<img width="246" height="198" alt="image" src="https://github.com/user-attachments/assets/72683946-7df0-4dc0-a280-7f96ff4c75aa" />

should have been this:

<img width="184" height="192" alt="image" src="https://github.com/user-attachments/assets/3fc804d9-da66-4cd0-bc51-9508e6f1f01a" />

but github displays that incorrectly.

Reply:
apart from the emoticons, you're right about your comment.

Reply:
done